model/lifecycle.go

```diff
@@ -877,6 +880,39 @@ func createTasksForBuild(creationInfo TaskCreationInfo) (task.Tasks, error) {
 	return tasks, nil
 }
 
+// checkUsersPatchTaskLimit takes in an input list of tasks that is set to get activated, and checks if they're
+// patch tasks, and that the request has been submitted by a user. If so, the maximum hourly patch tasks counter
+// will be incremented accordingly. the addExecutionTasks parameter indicates that execution tasks are included
+// as part of the input tasks param, otherwise we need to account for them.
+func checkUsersPatchTaskLimit(requester, username string, addExecutionTasks bool, tasks ...task.Task) error {
+	// we only care about patch tasks that are to be activated by an actual person
+	if !evergreen.IsPatchRequester(requester) || evergreen.IsSystemActivator(username) {
```

Comment:
Did we intend for this to also limit commit queue and merge queue tasks? (Since IsPatchRequester checks that)

Reply:
(and similarly are we considering API users to be real users here, in which case maybe we change the wording to "actual user")

Reply:
I think it's fair to consider commit queue tasks in this limitation, since they're adjacent to PR patches which we're also checking. Per other comment I'm opting to not consider API users in these limits, updated wording.

Reply:
I think commit queue patches shouldn't be blocked by this -- that could result in a P1 level incident essentially.

Reply:
Ok, that's fair enough, I'm on board. Changing